lib/spack/spack/test/cc.py

```diff
@@ -170,12 +171,14 @@ def dump_mode(cc, args):
         return cc(*args, output=str).strip()
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Not yet implemented on windows")
```

Comment:
Module-level skip. Also to be clear: can you do this in all files where applicable? Even if there are only 2 test in the file, I recommend a module-level skip rather than individually skipping them.

Reply:
Agreed. Will go though the tests and handle these situations.